.github/src/Main.java

```diff
@@ -0,0 +1,43 @@
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Scanner userInput = new Scanner(System.in);
+
+        double number = userInput.nextDouble();
+        int myLuckyNumber = 29;
+
+        System.out.println("My lucky number is.: " + myLuckyNumber);
+
+        boolean wholeNumber = false;
```

Comment:
was genauso gehen würde und vlt etwas schöner ist: 
`boolean wholeNumber = Math.floor(number) == number;`
Wenn du einen Ausdruck in einem if verwendest, dann ist das ja ein boolscher wert - also true oder false. und den Wert kannst du natürlich direkt einer boolschen variable zuweisen, dann sparst du dir hier das if und das ganze ist etwas leichter zu lesen: 

```
boolean wholeNumber = number == Math.floor(number); 
System.out.println("Number is whole.: " + wholeNumber);
```

Ist kürzer - weniger code ist immer besser 😉